client/src/controllers/ProgressController.ts

```diff
@@ -78,10 +95,7 @@ export class ProgressController extends Controller {
               labelElement.setAttribute(`data-${identifier}-target`, 'label');
             }
 
-            button.setAttribute(
-              `data-${identifier}-duration-seconds-value`,
-              '30',
-            );
+            button.setAttribute(`data-${identifier}-duration-value`, '10000');
```

Comment:
```suggestion
            button.setAttribute(`data-${identifier}-duration-value`,  `${30_000}`);
```

This may read better. Also, should be 30 seconds.

Reply:
update- this line is not actually needed at all.